pkg/asset/installconfig/aws/platform.go

```diff
@@ -9,12 +9,13 @@ import (
 	"github.com/sirupsen/logrus"
 	survey "gopkg.in/AlecAivazis/survey.v1"
 
-	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/aws"
+	"github.com/openshift/installer/pkg/version"
```

Comment:
probably want to make it clear in the commit message and description that it is set to the payload architecture only if the binary is extracted from the payload and not custom built. this might be misleading if someone looks at this commit and assumes that it always matches the payload in general

Reply:
when you build the installer yourself, then you're probably already overriding the payload at create cluster time anyway.  at that point you're already on your own, so having to possibly edit the install-config shouldn't be onerous.